src/scripts/librarymenu.js

```diff
@@ -155,297 +302,669 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
+            });
+        });
+    }
+
+    var navDrawerElement;
+    var navDrawerScrollContainer;
+    var navDrawerInstance;
+    var mainDrawerButton;
+    var headerHomeButton;
+    var currentDrawerType;
+    var pageTitleElement;
+    var headerBackButton;
+    var headerUserButton;
+    var currentUser;
+    var headerSettingsButton;
+    var headerCastButton;
+    var headerSearchButton;
+    var enableLibraryNavDrawer = !layoutManager.tv;
+    var skinHeader = document.querySelector(".skinHeader");
+    var requiresUserRefresh = true;
+    var lastOpenTime = new Date().getTime();
     window.LibraryMenu = {
         getTopParentId: getTopParentId,
-        onHardwareMenuButtonClick: function() {
-            toggleMainDrawer()
+        onHardwareMenuButtonClick: function () {
+            toggleMainDrawer();
         },
-        setTabs: function(type, selectedIndex, builder) {
-            require(["mainTabsManager"], function(mainTabsManager) {
-                type ? mainTabsManager.setTabs(viewManager.currentView(), selectedIndex, builder, function() {
-                    return []
-                }) : mainTabsManager.setTabs(null)
-            })
+        setTabs: function (type, selectedIndex, builder) {
+            require(["mainTabsManager"], function (mainTabsManager) {
+                if (type) {
+                    mainTabsManager.setTabs(viewManager.currentView(), selectedIndex, builder, function () {
+                        return [];
+                    });
+                } else {
+                    mainTabsManager.setTabs(null);
+                }
+            });
         },
-        setDefaultTitle: function() {
-            pageTitleElement || (pageTitleElement = document.querySelector(".pageTitle")), pageTitleElement && (pageTitleElement.classList.add("pageTitleWithLogo"), pageTitleElement.classList.add("pageTitleWithDefaultLogo"), pageTitleElement.style.backgroundImage = null, pageTitleElement.innerHTML = ""), document.title = "Jellyfin"
+        setDefaultTitle: function () {
+            if (!pageTitleElement) {
+                pageTitleElement = document.querySelector(".pageTitle");
+            }
+
+            if (pageTitleElement) {
+                pageTitleElement.classList.add("pageTitleWithLogo");
+                pageTitleElement.classList.add("pageTitleWithDefaultLogo");
+                pageTitleElement.style.backgroundImage = null;
+                pageTitleElement.innerHTML = "";
+            }
+
+            document.title = "Jellyfin";
         },
-        setTitle: function(title) {
-            if (null == title) return void LibraryMenu.setDefaultTitle();
-            "-" === title && (title = "");
+        setTitle: function (title) {
+            if (null == title) {
+                return void LibraryMenu.setDefaultTitle();
+            }
+
+            if ("-" === title) {
+                title = "";
+            }
+
             var html = title;
```

Comment:
I can see no difference from this point with `setDefaultTitle()`

Reply:
Not sure what you are hinting at here.

Reply:
I'm saying that this function seems to be a copy-paste of another one. But that should be cleaned up in subsequent PR anyway.